### ISSUE TITLE
Support for deletes

### ISSUE THREAD

**Issue by bluestreak01** [questdb/roadmap#10]
## Summary

This is a sister problem to #7. We aim to support `delete` SQL in similar use-cases.

## Expected outcome

The following syntax will be supported

```
[ WITH with_query [, ...] ]
DELETE FROM table_name  [ [ AS ] alias ]
    [ USING from_item [, ...] ]
    [ WHERE condition ]
```

This issue depends on #9

**Comment by saurav100-coder** [comment]
Is QuestDb support 'post' crud operation?
if yes please explain how

**Comment by Roh-codeur** [comment]
@bluestreak01 awesome product! looks like [#9 ] is complete - do we know if we have made any progress on this please? this is quite key to our adoption of QuestDB

ta!

**Comment by ghost** [comment]
Does anybody know how much this would take to implement? Is there a way we can help with it?

**Comment by 1185497683** [comment]
Is QuestDb support 'update' and 'delete' crud operation?
if yes please explain how

**Comment by chen-001** [comment]
I tested yesterday, Questdb still does not support DELETE. 
But UPDATE is supported. 
Examples for UPDATE are shown on (examples for UPDATE)[https://github.com/questdb/questdb/issues/163]

**Comment by chen-001** [comment]
I tested yesterday, Questdb still does not support DELETE.
But UPDATE is supported.
Examples for UPDATE are shown on [examples for UPDATE](https://github.com/questdb/questdb/issues/163)

**Comment by hokletrain** [comment]
Hi guys, any progress on this?

**Comment by mmahdium** [comment]
It's been postponed only to be removed from the roadmap. Is there still a plan to implement this, or is it being forgotten for now?
`UPDATE` was added as in #7, but still no `DELETE`?

**Comment by mmahdium** [comment]
> It's been postponed only to be removed from the roadmap. Is there still a plan to implement this, or is it being forgotten for now? `UPDATE` was added as in [#7](https://github.com/questdb/roadmap/issues/7), but still no `DELETE`?

@bluestreak01 Still no updates on this?
